specification/purview/data-plane/Azure.Analytics.Purview.DataMap/preview/2023-10-01-preview/purviewdatamap.json

```diff
@@ -1551,6 +1551,151 @@
         }
       }
     },
+    "/entity/bulk/delete": {
+      "delete": {
+        "tags": [
+          "Entity"
+        ],
+        "consumes": [
+          "multipart/form-data"
+        ],
+        "description": "Delete entities with hierarchy",
+        "operationId": "Entity_DeleteWithHierarchy",
+        "x-ms-examples": {
+          "Entity_DeleteWithHierarchy": {
+            "$ref": "./examples/Entity_DeleteWithHierarchy.json"
+          }
+        },
+        "parameters": [
+          {
+            "name": "guid",
+            "in": "query",
+            "description": "An array of GUIDs of entities to delete.",
+            "required": true,
+            "type": "array",
+            "items": {
+              "type": "string"
+            },
+            "collectionFormat": "multi",
+            "x-ms-client-name": "guids"
+          },
+          {
+            "name": "parallelCount",
+            "in": "query",
+            "description": "",
+            "required": false,
+            "type": "string",
+            "enum": [
+              "PARALLEL_COUNT_4",
+              "PARALLEL_COUNT_8",
+              "PARALLEL_COUNT_16",
+              "PARALLEL_COUNT_32",
+              "PARALLEL_COUNT_64"
+            ],
+            "x-ms-enum": {
+              "name": "parallelCount",
```

Comment:
Thanks. it is only for bulk delete API. Description udpated.

Reply:
Sure. This may get back from SDK review, that a `ParallelCount` (if you get to typespec) would be too generic.

So far this is fine for Swagger and preview.